src/player.js

```diff
@@ -444,6 +444,7 @@ export default class Player extends FakeEventTarget {
       this._configureOrLoadPlugins(config.plugins);
       this._maybeCreateAdsController();
       this.reset();
+      this.handleAdsWithMSE();
```

Comment:
this seems like a player instance level config, and not source level config, so this is not a proper location